src/main/java/com/arturfrimu/exercisesback/service/ExerciseService.java

```diff
@@ -0,0 +1,5 @@
+package com.arturfrimu.exercisesback.service;
+
+public interface ExerciseService extends SetExerciseConfigurationService, GenerateExerciseService,ListAllExercisesService,
+        FindExerciseService,FindPercentageExerciseService,ExerciseManagementService {
```

Comment:
Nu este nevoie sa combini toate interfetele în una singura

Trebuie fiecare interfata sa aiba si propria sa implementare

Apoi fiecare interfata v-a fi injectata pe cont propriu în controller

poti face merge la acest PR acum, dar sa faci ce am scris mai sus in alt branch, BUNA TREABĂ 💪🏻